### ISSUE TITLE
Getting issue of libarrow.so not found.

### ISSUE THREAD

**Issue by mourya09** [heavyai/heavydb#518]
Hi Team,

I am trying to built and start the server from source code. After installing all the requisite and then I am trying to start the server. It is giving me the following error:
```
[admin@admin build]$ sudo ../startomnisci
startomnisci 3032 running
Backend TCP:  localhost:6274
Backend HTTP: localhost:6278
Frontend Web: localhost:6273
Calcite TCP:  localhost:6279
- omnisci_server 3044 started
**./bin/omnisci_server: error while loading shared libraries: libarrow.so.13: cannot open shared object file: No such file or directory**
startomnisci 3032 exited
```
Environment details:
It is a VM with 32 Gb RAM and 500Gb hard disk without any GPU. So I built without CUDA library.
OS is Centos 7.

Please help me in rectifying the error.

Thanks
Praveen Kumar

**Comment by cdessanti** [comment]
Hi @mourya09 ,

it's likely you compiled the server with the flag PREFER_STATIC_LIBS set to off, so you should  recompile link the library to the server in a static fashion or run the mapd-deps.sh you used when you compile the server.

**Comment by mourya09** [comment]
Hi @cdessanti 

Let me try again and will post the result.

Thanks
Praveen Kumar

**Comment by psilvest** [comment]
This issue has been inactive for a while, and the project has changed substantially. If this still occurs on the latest release, please add reproduction steps and environment details.
